src/datachain/lib/file.py

```diff
@@ -115,22 +115,24 @@ class File(DataModel):
     parent: str = Field(default="")
     name: str
     size: int = Field(default=0)
-    version: str = Field(default="")
     etag: str = Field(default="")
+    version: str = Field(default="")
     is_latest: bool = Field(default=True)
-    last_modified: datetime = Field(default=TIME_ZERO)
-    location: Optional[Union[dict, list[dict]]] = Field(default=None)
     vtype: str = Field(default="")
+    location: Optional[Union[dict, list[dict]]] = Field(default=None)
+    last_modified: datetime = Field(default=TIME_ZERO)
 
     _datachain_column_types: ClassVar[dict[str, Any]] = {
         "source": String,
         "parent": String,
         "name": String,
-        "version": String,
-        "etag": String,
         "size": Int,
+        "etag": String,
+        "version": String,
+        "is_latest": Boolean,
         "vtype": String,
         "location": JSON,
+        "last_modified": DateTime,
```

Comment:
Should we add `owner_id` and `owner_name` as well?

Reply:
TBH, I'm not really sure what this change does. `File._datachain_column_types` is used in a few unrelated places for things that seem mostly irrelevant.
BTW, `owner_id` and `owner_name` are deprecated and should be removed soon, see #36.

Reply:
By default, DataChain starts with the signal schema from `File._datachain_column_types`.

https://github.com/iterative/datachain/blob/f7fdf6b70f323c74f6bbf8bfb9ae08b66755d00c/src/datachain/lib/dc.py#L209

https://github.com/iterative/datachain/blob/f7fdf6b70f323c74f6bbf8bfb9ae08b66755d00c/src/datachain/query/dataset.py#L1102


https://github.com/iterative/datachain/blob/f7fdf6b70f323c74f6bbf8bfb9ae08b66755d00c/src/datachain/lib/dc.py#L218


So, this is just adding them to `SignalSchema` so that `map` can use it to pass to the function.